workitem/workitem_repository.go

```diff
@@ -947,10 +947,19 @@ func (r *GormWorkItemRepository) getAllIterationWithCounts(ctx context.Context,
 	var allIterations []uuid.UUID
 	db.Pluck("id", &allIterations)
 	var res []WICountsPerIteration
-	db = r.db.Table(workitemTableName).Select(`iterations.id as IterationId, count(*) as Total,
-			count( case fields->>'system.state' when 'closed' then '1' else null end ) as Closed`).Joins(`left join iterations
-			on fields@> concat('{"system.iteration": "', iterations.id, '"}')::jsonb`).Where(`iterations.space_id = ?
-			and work_items.deleted_at IS NULL`, spaceID).Group(`IterationId`).Scan(&res)
+	db = r.db.Table(workitemTableName).Select(`
+		iterations.id AS IterationId,
+		count(*) AS Total,
+		count(1) FILTER (WHERE fields->>'system.state' = 'closed') AS Closed
```

Comment:
Is `count(1)` any different from `count(*)`?  I was reading about it here https://www.postgresql.org/docs/9.5/static/functions-aggregate.html and it looks like the result would always be same in this case.

Reply:
Well, I find much more intuitive to use `count(1)` somehow.

Reply:
Can we change it to `count(*)` please? Everybody might not know about `count(1)`. I had to google a bit :)

Reply:
I that fits you better, yes.

Reply:
Done cf8d8d5